Update nodejs version to 12 - Closes #429 (#439)
LiskHQ/lisk-docs: modules/ROOT/partials/install-node.adoc

NVM is a bash script which enables the management of multiple active Node.js versions.

. To install nvm, please follow these https://github.com/creationix/nvm#install--update-script[instructions].

> . Please install the latest version of Node.js using NVM, and execute the command listed below:

Reviewer: ```suggestion
. Install the latest LTS version of Node.js:
```

[source,bash]
----

> nvm install 12.14.1

Author: ```suggestion
nvm install 12.15.0
```

[source,bash]
----

> brew install node@12.14.1

Author: ```suggestion
brew install node@12.15.0
```
